lib/grape/exceptions/base.rb

```diff
@@ -51,12 +51,12 @@ def resolution(key, attributes)
 
       def translate_attributes(keys, **options)
         keys.map do |key|
-          translate("#{BASE_ATTRIBUTES_KEY}.#{key}", default: key, **options)
+          translate("#{BASE_ATTRIBUTES_KEY}.#{key}", default: key.to_s, **options)
         end.join(', ')
       end
 
       def translate_attribute(key, **options)
-        translate("#{BASE_ATTRIBUTES_KEY}.#{key}", default: key, **options)
+        translate("#{BASE_ATTRIBUTES_KEY}.#{key}", default: key.to_s, **options)
```

Comment:
Maybe I am nitpicking, but feels like we should do this inside `translate`, no?

Reply:
It is generally better. But I'm not sure if there is any need to pass an alternative symbol key as value of :default in other cases, though, there isn't any example now.

Reply:
It is generally better. But I'm not sure if there is any need to pass an alternative symbol key as value of :default in other cases, though, there isn't any example now.